src/main/resources/view/DarkTheme.css

```diff
@@ -161,6 +166,15 @@
     text-align: center;
 }
 
+body, html {
+    height: 99%;
```

Comment:
why 99% and not 100% :>

Reply:
Somehow 100% will result in the page having a scroll bar in our normal setting of the GUI, but 99% doesn't